generator/app.go

```diff
@@ -215,15 +220,15 @@ func createGroupReadme(groups []Group, prefix string) error {
 
 		fmt.Printf("Generating %s/README.md\n", group.Dir)
 
-		outputDir := filepath.Join(baseOutputDir, group.Dir)
+		outputDir := filepath.Join(baseGeneratorDir, group.Dir)
 		if err := createDirIfNotExists(outputDir); err != nil {
 			return err
 		}
 
 		group.SetupGitHubTeams(prefix)
 
 		outputPath := filepath.Join(outputDir, indexFilename)
-		readmePath := fmt.Sprintf("%s_%s", prefix, readmeTemplate)
+		readmePath := fmt.Sprintf("%s/%s_%s", filepath.Join(baseGeneratorDir, templateDir), prefix, readmeTemplate)
```

Comment:
nit: how about `filepath.Join(baseGeneratorDir, templateDir, fmt.Sprintf("%s_%s", prefix, readmeTemplate))`?

Reply:
I like that much better!